FaceID/faceID.py

```diff
@@ -235,15 +235,21 @@ def getImagesAndLabels(path):
             imagePaths = [os.path.join(path,f) for f in os.listdir(path)]
             faceSamples = []
             ids = []
+
             for imagePath in imagePaths:
-                PIL_img = Image.open(imagePath).convert('L')
-                img_numpy = np.array(PIL_img,'uint8')
-                id = int(os.path.split(imagePath)[-1].split("_")[0])
-                print(id)
-                faces = detector.detectMultiScale(img_numpy)
-                for (x,y,w,h) in faces:
-                    faceSamples.append(img_numpy[y:y+h,x:x+w])
-                    ids.append(id)
+                picName = os.path.split(imagePath)[-1] 
+                ext = os.path.split(imagePath)[-1].split(".")[1]
+                
+                if ext == "jpg":
+                    id = int(picName.split("_")[0])
+                    PIL_img = Image.open(imagePath).convert('L')
+                    img_numpy = np.array(PIL_img,'uint8')
+                    faces = detector.detectMultiScale(img_numpy)
+                    print(": ", id)
+                    for (x,y,w,h) in faces:
+                        faceSamples.append(img_numpy[y:y+h,x:x+w])
+                        ids.append(id)
+            print("")
```

Comment:
Is this `print` needed?

Reply:
It gives some space after the training data is printed. I could remove the training data prints if we don't need them